src/L2VoteAggregator.sol

```diff
@@ -118,6 +122,7 @@ abstract contract L2VoteAggregator {
 
     bytes memory proposalCalldata = abi.encode(proposalId, vote.against, vote.inFavor, vote.abstain);
     _bridgeVote(proposalCalldata);
+    emit VoteBridged(proposalId, vote.against, vote.inFavor, vote.abstain);
```

Comment:
Do you think having the sequence of the message would be helpful at all?

Reply:
sequence is emitted in the related wormhole event